pkg/invariants/kubeapiserver/disruptionlegacyapiservers/invariant.go

```diff
@@ -144,6 +147,28 @@ func newDisruptionCheckerForOAuthCached(adminRESTConfig *rest.Config) (*disrupti
 
 func (w *availability) StartCollection(ctx context.Context, adminRESTConfig *rest.Config, recorder monitorapi.RecorderWriter) error {
 	var err error
+
+	kubeClient, err := kubernetes.NewForConfig(adminRESTConfig)
+	if err != nil {
+		return err
+	}
+	namespaceExists, err := exutil.CheckNamespaceExists(kubeClient, "openshift-apiserver")
+	if err != nil {
+		return fmt.Errorf("unable to check namespace existence: %v", err)
+	}
+	if !namespaceExists {
+		w.notSupportedReason = "namespace openshift-apiserver not present"
+		return nil
```

Comment:
this looks incorrect.  You only need to skip the apiservers not present, rigth?  the kube-api should still be watched.